Sources/Commands/SwiftTool.swift

```diff
@@ -315,26 +326,31 @@ public class SwiftTool {
     /// The file system in use
     let fileSystem: FileSystem
 
+    private let wantsMultipleTestProducts: Bool
+    private let wantsREPLProduct: Bool
```

Comment:
perhaps we can capture these in a configuration struct, so that the initializer signature can grow over time as we have more of them?

Reply:
we have `GlobalOptions` which comes from argument parsing, so maybe we add another one eg `InternalOptions`, or we even find a way to put all of it into `GlobalOptions`

Reply:
Good point, maybe that can also simply the `SwiftCommand` protocol, instead of individual computed properties we can offer a single one that provides the custom configuration.

Reply:
I actually like the idea of adding this to GlobalOptions as configuration that does not come from argument parsing